setup.py

```diff
@@ -67,7 +76,7 @@
             'pyobjc-core>=3.0.3',
             'pyobjc-framework-Cocoa>=3.0.3',
             'pyobjc-framework-Quartz>=3.0.3',
-            'wxPython>=2.8',
+            'wxPython>=3.0',
```

Comment:
I am on Arch Linux. Looks like it's available on Ubuntu starting with Vivid (15.04), so I think we're good.